client/lib/importer/importer-config.tsx

```diff
@@ -294,6 +304,90 @@ function getConfig( {
 		weight: 0,
 	};
 
+	importerConfig.blogroll = {
+		engine: 'blogroll',
+		key: 'importer-type-blogroll',
+		type: 'url',
+		priority: 'secondary',
+		title: 'Blogroll',
+		icon: 'blogroll',
+		description: '',
+		uploadDescription: '',
+		weight: 0,
+	};
+
+	importerConfig.ghost = {
```

Comment:
We should remove Ghost from the list of available importers: paYKcK-4P7-p2#comment-3559

Reply:
Done!